src/ResourceManager/Compute.ManagedService/Commands.Compute.ManagedService/help/ConvertTo-AzureRmVhd.md

```diff
@@ -0,0 +1,119 @@
+---
+external help file: 
+Module Name: 
+online version: 
+schema: 2.0.0
+---
+
+# ConvertTo-AzureRmVhd
+
+## SYNOPSIS
+Convert Hyper-V VM to Azure supported virtual hard disk files
+
+## SYNTAX
+
+```
+ConvertTo-AzureRmVhd -HyperVVMName <String> -ExportPath <String> [-HyperVServer <String>] [-Force] [-AsJob]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Convert Hyper-V VM to Azure supported virtual hard disk files
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> ConvertTo-AzureRmVhd -HyperVVMName 'test' -ExportPath '.';
```

Comment:
@huangpf would you please add the corresponding output for this cmdlet execution?

Reply:
OK. But just for curiosity, I check the documentation and examples for ```Convert-Vhd```, and they don't seem to bother to include the output. :-p